### PR TITLE
{system}[system/system] CUDA v11.1.1, CUDAcore v11.1.1, gcccuda v2020b

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Or are we waiting for some newer CUDA for some compiler compatibility stuff?

File: easybuild/easyconfigs/c/CUDAcore/CUDAcore-11.1.1.eb
Comment (boegel) [inline easybuild/easyconfigs/c/CUDAcore/CUDAcore-11.1.1.eb:16]: This should be extended, to also cover the checksum for the `ppc64le` installer, like we're doing for CUDA 11.0:

```suggestion
checksums = [
    {
        'cuda_%%(version)s_%s_linux.run' % local_nv_version:
            '3eae6727086024925ebbcef3e9a45ad379d8490768fd00f9c2d8b6fd9cd8dd8f',
        'cuda_%%(version)s_%s_linux_ppc64le.run' % local_nv_version:
            '023e571fe26ee829c98138dfc305a92279854aac7d184d255fd58c06c6af3c17',
    }
]
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
> Or are we waiting for some newer CUDA for some compiler compatibility stuff?

I was mostly waiting until `foss/2020b` was merged (which is done now, see #11489).

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
easybuild1.novalocal - Linux centos linux 8.2.2004, POWER, IBM pSeries (emulated by qemu) (power8le), Python 3.6.8
See https://gist.github.com/88fe3de9c6f23d94c386fae65959c88e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8016bcffa8e0619f21e0d7e266250e80 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3626.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/dd506df61680b5edfc3256e41fe8223c for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**
